rclcpp/include/rclcpp/node_impl.hpp

```diff
@@ -196,44 +196,9 @@ Node::create_service(
   rmw_service_t * service_handle = rmw_create_service(
     this->node_handle_, service_type_support_handle, service_name.c_str());
 
-  auto serv = service::Service<ServiceT>::make_shared(
-    service_handle,
-    service_name,
-    callback);
+  auto serv = create_service_internal<ServiceT>(service_handle, service_name,
+      callback);
```

Comment:
You are right - it passes. That looks like a bug in uncrustify to me then...

Since we want 2 space indentation can you please change it to the follow to pass uncrustify as well as fit with our style:

      auto serv = create_service_internal<ServiceT>(
        service_handle, service_name, callback);